packages/plugin-core/src/commands/Doctor.ts

```diff
@@ -67,6 +96,71 @@ export class DoctorCommand extends BasicCommand<CommandOpts, CommandOutput> {
         });
         break;
       }
+      case DoctorActions.CREATE_MISSING_LINKED_NOTES: {
+        const notes = engine.notes;
+        // pick out wild wikilinks
+        let wildWikiLinks = [] as DLink[];
+        _.forEach(notes, (note) => {
+          const links = note.links;
+          if (_.isEmpty(links)) {
+            return false;
+          }
+          const wsRoot = DendronWorkspace.wsRoot();
+          wildWikiLinks = wildWikiLinks.concat(
+            _.filter(links, (link) => {
+              if (link.type != "wiki") {
+                return false;
+              }
+              const noteExists = NoteUtils.getNoteByFnameV5({
+                fname: link.to!.fname as string,
+                vault: note.vault,
+                notes: notes,
+                wsRoot: wsRoot,
+              }) as NoteProps;
+              return !noteExists;
+            })
+          );
+          return true;
+        });
+
+        // pick out unique candidate notes to create
+        const uniqueCandidates = _.map(
+          _.uniqBy(wildWikiLinks, "to.fname"),
+          (link) => {
+            return {
+              fname: link.to!.fname,
+              vault: link.from.vault,
+            };
+          }
+        ) as NoteProps[];
+        if (uniqueCandidates.length > 0) {
+          // show preview before creating
+          await this.showPreview(uniqueCandidates);
+          const options = ["proceed", "cancel"];
+          const shouldProceed = await VSCodeUtils.showQuickPick(options, {
```

Comment:
check for `undefined` here, it means the user pressed escape in which case we should cancel the command

Reply:
wouldn't `shouldProceed !== "proceed"` check for `undefined` as well? If I run press escape it will cancel the command and show the information message correctly.

Reply:
your right, i missed that!